src/main/java/com/exactpro/th2/healer/cfg/HealerConfiguration.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.healer.cfg;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.Objects;
+
+public class HealerConfiguration {
+
+    @JsonProperty
+    private String name;
+    @JsonProperty
+    private String version;
+    @JsonProperty
+    private int initialCacheCapacity;
+    @JsonProperty
+    private int maxCacheCapacity;
+
+    public HealerConfiguration(String name, String version, int initialCacheCapacity, int maxCacheCapacity) {
```

Comment:
You won't be able to create this object in ObjectMapper. You need to add annotation `@JsonProperty` with parameter names for each parameter in the constructor. Also, you need to annotate the constructor with `@JsonCreator` annotation. Useful link: https://www.baeldung.com/jackson-annotations#1-jsoncreator